cvat/apps/dataset_manager/bindings.py

```diff
@@ -267,6 +269,7 @@ class Frame(NamedTuple):
         shapes: Sequence[CommonData.Shape]
         labels: Sequence[CommonData.Label]
```

Comment:
```suggestion
        labels: Mapping[int, CommonData.Label]
```
This field actually seems to be unused, as expected.